src/AcceptanceTests/Routing/When_using_single_namespace.cs

```diff
@@ -0,0 +1,105 @@
+namespace NServiceBus.Azure.Transports.WindowsAzureServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Support;
+    using AzureServiceBus;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_using_single_namespace : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_append_namespace_name_to_reply_address_when_using_names()
+        {
+            var runSettings = new RunSettings();
+            runSettings.TestExecutionTimeout = TimeSpan.FromMinutes(1);
+
+            var ctx = new AzureServiceBusTransportConfigContext();
+            ctx.Callback = (endpointName, extensions) =>
+            {
+                var connectionString = Environment.GetEnvironmentVariable("AzureServiceBusTransport.ConnectionString");
+                extensions.ConnectionString(connectionString);
+                extensions.UseNamespaceNamesInsteadOfConnectionStrings();
+            };
+
+            runSettings.Set("AzureServiceBus.AcceptanceTests.TransportConfigContext", ctx);
+
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<SourceEndpoint>(b =>
+                {
+                    b.When(async (bus, c) =>
+                    {
+                        var sendOptions = new SendOptions();
+                        sendOptions.SetDestination("usingsinglenamespace.targetendpoint");
```

Comment:
Just a question - is there a way to get the endpoint name w/o hard-coding a string? We're doing this at several places. Not the end of the world, but curious since ATTs are doing so :)